test/apps/iodemo/io_demo.cc

```diff
@@ -1518,28 +1553,36 @@ class DemoClient : public P2pDemoCommon {
         }
     }
 
+    static void log_disconnect_info(const server_info_t& server_info,
```

Comment:
why need a separate function?disconnect_server() is already very short

Reply:
removed